src/Nethermind/Nethermind.Blockchain/Tracing/BlockReceiptsTracer.cs

```diff
@@ -35,7 +35,7 @@ public class BlockReceiptsTracer : IBlockTracer, ITxTracer, IJournal<int>, ITxTr
 
     public void MarkAsSuccess(Address recipient, GasConsumed gasSpent, byte[] output, LogEntry[] logs, Hash256? stateRoot = null)
     {
-        _txReceipts.Add(BuildReceipt(recipient, gasSpent.SpentGas, StatusCode.Success, logs, stateRoot));
+        _txReceipts.Add(BuildReceipt(recipient, gasSpent.SpentGas, StatusCode.Success, logs, gasSpent.PolicyData, stateRoot));
```

Comment:
just pass `gasSpent` to `BuildReceipt` and `BuildFailedReceipt`, so we can ignore `PolicyData`